docling-server/main.py

```
@@ -0,0 +1,385 @@
+            os.close(fd)
+            raise
+        
+        # Process in thread pool (blocking operation)
+        loop = asyncio.get_running_loop()
+        result = await loop.run_in_executor(None, process_document_sync, tmp_path)
+        
+        return JSONResponse(content=result)
+        
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)